### ISSUE TITLE
Offline Sync Issue

### ISSUE THREAD

**Issue by minwka** [clipto-pro/Desktop#132]
Any changes made on mobile app does not sync back to desktop app upon launch. So the desktop app becomes out of date if it was not online during changes. Out of date notes are only synced when individual notes are edited from source, one by one.

Thank you.

**Comment by atrashler** [comment]
> Any changes made on mobile app does not sync back to desktop app upon launch. So the desktop app becomes out of date if it was not online during changes. Out of date notes are only synced when individual notes are edited from source, one by one.
> 
> Thank you.

The sync should work instantly whenever the internet is available.
Could you please write the versions of the Mobile and Desktop apps?

**Comment by minwka** [comment]
7.2.6 on desktop, 7.3.8-release on Android.
(Edit: It may be an issue with the VPN one my phone. Just made a clean direct connection and everything seems to catch up to each other.)

**Comment by d0x360** [comment]
same issue here, in fact i just updated the desktop version and a dialog popped up but i didnt get to read it because i hit space by accident.  Its also not copying thing i copy on Win10

**Comment by d0x360** [comment]
So i updated to version ending in ,17, sorry im not on pc anymore and it fixed it

**Comment by atrashler** [comment]
> 7.2.6 on desktop, 7.3.8-release on Android.
> (Edit: It may be an issue with the VPN one my phone. Just made a clean direct connection and everything seems to catch up to each other.)

Yes. VPN can block full-duplex connections (depends on the VPN provider and the cost of it). BTW, which VPN provider is it?

**Comment by minwka** [comment]
Not a proper VPN. It's called Tailscale and I use it to access my computer remotely.

**Comment by kittiesauce** [comment]
My android Clip-to notes are not syncing with my desktop PC at work. I do not have a VPN. I have 66 notes on my phone and only 4 of them have synced to my desktop. I have tried signing out and signing back in but the notes are not there. Strangely though, all of my TAGS have synced - but for each tag it shows 0 notes, as opposed to the many notes for each tag I have on my phone.
